test-installation.bat

```diff
@@ -0,0 +1,58 @@
+@echo off
+
+REM Prompt the user to drag and drop the location of the Llama-based model (.bin) and read it from standard input
+echo --GPT-LLAMA.CPP TEST INSTALLATION SCRIPT LAUNCHED--
+echo PLEASE MAKE SURE THAT A LOCAL GPT-LLAMA.CPP SERVER IS STARTED. OPEN A SEPARATE TERMINAL WINDOW START IT.
+set /p modelpath=Please drag and drop the location of your Llama-based Model (.bin) here and press enter:
+
+REM Check if the file exists
+if not exist "%modelpath%" (
+    echo Error: The file does not exist. Please make sure you have provided the correct path.
+    pause
+    exit /b 1
+)
+
+REM Check if the file has a .bin extension
+set "ext=%modelpath:~-4%"
+if not "%ext%" == ".bin" (
+    echo Error: The file extension is not .bin. Please make sure you have provided the correct file.
+    pause
+    exit /b 1
+)
+
+set "validpath=llama.cpp\models\"
```

Comment:
this still didn't work for me. also please update the error message to remove `\`